infrastructure/time/duration.hh

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <stdint.h>  // uint8_t etc
+
+namespace jet {
+
+class Duration {
```

Comment:
Might suggest doing something like this: https://github.com/jpanikulam/experiments/blob/master/estimation/time_point.hh